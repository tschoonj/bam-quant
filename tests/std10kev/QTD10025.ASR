File: C:\xrfdaten\2014KW30\chris\std10kev\std10025.spe
Start: 21.07.2014 15:59:00
Stop: 21.07.2014 16:00:41
BESSY: 298.2900416 298.1868291
IONI: 1.448228E-10 1.446374E-10
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 80605
Run: 100000
Dead: 19
ICR: 64650
OCR: 0
Accumulated BESSY: 59685.9912743
Accumulated Ioni: 2.89754638E-08
N Accumulation: 200
Normfactor:       85632302.
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Mn
exp_x 132.020401000977 132.0204
exp_y 150.265808105469 150.2658
exp_z 53.8899993896484 53.8901
exp_w 135 135
exp_Tisch 871.55 871.549999496
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 10 10.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 0.03
$PEAKS:   
      1
  25   1      5.895   4463208.      2113.     133.19
