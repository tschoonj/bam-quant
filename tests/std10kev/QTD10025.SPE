$DATA:
0 2000
18703
12123
5250
1566
301
64
10
2
7
1
4
0
1
0
0
3
152
1177
1446
1328
1380
1224
1164
1146
1137
1139
1041
1034
1001
989
989
969
935
865
922
879
819
873
840
854
786
816
806
808
748
704
708
726
708
676
709
697
720
723
720
671
655
690
641
687
653
668
694
633
651
633
678
669
622
603
656
670
627
633
696
649
617
571
592
565
611
587
593
568
621
603
549
594
615
586
621
579
606
613
597
589
581
595
633
680
654
648
756
666
658
639
589
599
619
608
597
591
551
560
580
602
552
533
564
571
579
514
585
568
527
601
596
647
650
645
647
643
600
544
594
545
600
619
599
597
601
620
701
767
922
1021
1107
1139
975
906
741
631
595
596
559
581
600
618
661
643
681
640
617
603
566
601
590
618
570
586
602
580
562
540
589
590
577
587
607
601
618
661
666
698
708
706
614
663
653
642
649
641
650
705
699
691
747
770
769
950
1185
1567
1986
2542
3015
3281
3232
2711
2183
1724
1215
911
842
783
725
775
728
729
800
739
813
822
771
834
784
905
812
857
911
901
961
1004
1135
1189
1194
1139
1192
1148
1069
1081
1032
1098
1046
1072
1083
1112
1073
1077
1112
1188
1136
1168
1194
1263
1175
1196
1236
1238
1307
1263
1223
1257
1251
1219
1331
1390
1440
1502
1500
1690
1735
1782
1957
2117
2169
2373
2517
2890
3130
3566
4327
5327
7717
13243
25594
50660
95030
167055
268133
385077
496606
574062
591010
544507
447454
329141
216733
128284
68090
32744
14992
6730
3298
2070
1663
1639
1600
1679
1784
2091
2482
3019
4145
5874
9581
15427
24975
37496
52531
67381
78899
83316
79791
68849
53944
37910
24091
13843
7225
3459
1547
686
302
145
85
80
75
53
60
59
39
44
42
45
63
55
61
65
43
54
44
46
37
34
30
25
29
25
33
13
20
22
26
24
19
20
20
22
23
25
17
27
20
13
10
12
15
14
13
13
14
23
14
13
18
15
6
12
20
13
16
20
29
18
34
37
31
37
27
30
28
23
23
25
19
19
18
14
19
14
9
19
15
8
19
19
18
11
23
23
34
46
59
75
133
143
147
130
155
139
132
78
67
45
26
31
18
16
21
18
19
18
11
11
16
15
10
18
19
11
12
12
15
8
5
13
8
10
10
11
14
12
13
14
12
19
19
17
27
29
28
29
29
44
47
31
43
45
41
34
51
60
77
86
105
127
152
176
234
260
331
384
375
458
420
429
451
364
313
270
208
159
92
73
61
40
27
24
15
7
8
12
14
4
10
8
7
14
19
12
14
15
9
20
17
17
13
14
16
16
19
10
10
7
10
15
12
10
11
14
18
14
7
14
10
21
15
11
10
11
14
8
11
15
21
14
21
14
22
17
21
15
21
21
27
17
35
24
31
35
40
53
92
158
248
375
569
814
1097
1350
1784
2062
2251
2334
2358
2105
1744
1469
1142
810
534
339
227
151
70
55
34
24
16
35
37
39
54
77
116
145
227
331
370
518
591
636
649
654
595
576
485
334
260
179
94
87
41
31
7
14
6
2
3
5
6
6
7
12
12
17
30
32
49
31
32
53
44
50
44
46
28
29
12
20
5
6
3
4
0
1
0
0
0
0
3
0
0
0
1
0
0
0
1
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
1
1
0
0
0
1
0
1
0
0
0
0
1
1
1
0
0
0
2
0
0
0
0
0
0
0
1
0
0
0
1
0
2
3
4
1
1
1
0
0
0
0
0
0
0
0
0
0
0
1
0
0
1
0
1
0
0
0
0
2
1
1
0
0
0
0
0
0
0
1
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
3
1
0
1
1
2
3
1
1
0
2
6
4
6
11
5
5
7
3
2
3
4
1
0
0
0
1
0
0
0
0
0
2
1
0
0
1
0
0
0
1
0
0
0
1
0
2
1
2
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
1
0
1
0
1
0
0
1
0
1
0
0
2
2
4
7
3
3
5
13
5
11
7
6
7
6
5
3
1
1
1
0
0
0
0
0
0
0
1
0
1
0
0
1
2
2
2
2
5
2
3
9
0
4
3
3
1
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
2
1
0
1
1
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
File: C:\xrfdaten\2014KW30\chris\std10kev\std10025.spe
Start: 21.07.2014 15:59:00
Stop: 21.07.2014 16:00:41
BESSY: 298.2900416 298.1868291
IONI: 1.448228E-10 1.446374E-10
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 80605
Run: 100000
Dead: 19
ICR: 64650
OCR: 0
Accumulated BESSY: 59685.9912743
Accumulated Ioni: 2.89754638E-08
N Accumulation: 200
Normfactor:       85632302.
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Mn
 
exp_x 132.020401000977 132.0204
exp_y 150.265808105469 150.2658
exp_z 53.8899993896484 53.8901
exp_w 135 135
exp_Tisch 871.55 871.549999496
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 10 10.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 0.03
$PEAKS:   
      1
  25   1      5.895   4463208.      2113.     133.19
