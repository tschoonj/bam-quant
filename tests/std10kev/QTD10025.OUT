     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:27:00  บ   
     บ  Spectrum: QTD10025.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 130 - 500        5 iterations done    บ   
     บ  ChiSquare =   332.8   last change =   .00%    lambda= 1.E-07   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ      -88.0 ๑   100.0   บ     -88.0 ๑      .9   บ   
     บ  GAIN (eV/ch)  บ     20.438 ๑   2.000   บ    20.438 ๑    .003   บ   
     บ det NOISE (eV) บ     -114.1 ๑    40.0   บ    -114.1 ๑    -3.0   บ   
     บ  FANO factor   บ       .275 ๑    .050   บ      .275 ๑    .004   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 Mn-Ka                           4463208. ๑   2113.           บ   
     บ    KA1         5.895   1.00000     4463208. ๑   2113.    133.2  บ   
     บ              292.741    146.87        1360.           8.96E-01  บ   
     บ                                                                 บ   
     บ  2 Mn-Kb                            670274. ๑    819.           บ   
     บ    KB1         6.491   1.00000      670274. ๑    820.     97.6  บ   
     บ              321.902    158.33         813.           9.21E-01  บ   
     บ                                                                 บ   
     บ  3 Pile_up                               0. ๑      0.           บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
